midi_app_controller/models/utils.py

```diff
@@ -60,7 +60,7 @@ def save_to(self, path: Path) -> None:
             yaml.safe_dump(self.dict(), f, default_flow_style=False)
 
 
-def find_duplicate(values: List[Any]) -> Optional[Any]:
+def find_duplicate(values: list[any]) -> Optional[any]:
```

Comment:
the same with `any`, i think the only builtin functions that have become aliases for generics are list, tuple, dict, set, frozenset and type